acq4/modules/DataManager/DataManagerModule.py

```diff
@@ -140,7 +136,11 @@ def currentDirChanged(self, name=None, change=None, args=()):
             with contextlib.suppress(HelpfulException):
                 newDir = self.manager.getCurrentDir()
 
-        has_images = newDir is not None and newDir.hasMatchingChildren(lambda f: f.shortName().endswith('.tif'))
+        has_images = newDir is not None and newDir.hasMatchingChildren(
+            lambda f: f.fileType() == "ImageFile" or f.shortName().startswith("ImageSequence_") or (
+                f.fileType() == "MetaArray" and 'pixelSize' in f.info()
```

Comment:
This is doing so of the same work as `representativeFramesForAllImages`, but it's not quite generalizeable...